src/features/SwapAssets/components/SwapQuote.tsx

```diff
@@ -112,7 +115,7 @@ export const SwapQuote = ({
       </Box>
       <Box>
         {!isExpired ? (
-          <TimeCountdown value={expiration} />
+          <TimeCountdown value={expiration} format={['days', 'hours', 'minutes']} />
```

Comment:
This is the default format, unnecesary.

Reply:
Indeed. I updated the default value to display seconds in order to facilitate component reuse.